src/app/pages/created-by-me/created-by-me.page.scss

```diff
@@ -17,4 +17,7 @@
 }
 .main-wrapper{
   margin : 20px 25px 0px 25px
+}
+ion-list{
```

Comment:
write a different class and over right, we should not modify the default tags directly